[14-20-18] [ENTRY]: [Nathan]
[14-20-19] [THREAD STARTED]: Singular Cyder Instance Ensurer
[14-20-19] [THREAD STARTED]: Splash Loader
[14-20-20] [THREAD STARTED]: Splash Animation
[14-20-20] [THREAD STARTED]: Cyder Start Secondary Subroutines
[14-20-20] [LOGIN]: [AUTOCYPHER ATTEMPT]
[14-20-20] [THREAD STARTED]: JVM Logger
[14-20-20] [ACTION]: Requested deletion of: C:\Users\Nathan\Documents\IntelliJava\Cyder\dynamic\tmp
[14-20-20] [SYSTEM_IO]: [JSON Saved] User was written to file: 7d7ebb00-81fe-329f-8a78-62f463d0b7da\Userdata.json
[14-20-20] [SYSTEM_IO]: Created temperatory file: 14-20-20.stacktrace
[14-20-20] [EXCEPTION]:
Exception origin: com.google.common.base.Preconditions.checkArgument(Preconditions.java:131)
From line: 131
Trace: java.lang.IllegalArgumentException
at com.google.common.base.Preconditions.checkArgument(Preconditions.java:131)
at cyder.utilities.FileUtil.fileContentsEqual(FileUtil.java:141)
at cyder.utilities.UserUtil.userJsonBackupSubroutine(UserUtil.java:244)
at cyder.utilities.UserUtil.setUserData(UserUtil.java:176)
at cyder.utilities.UserUtil.getterSetterFixer(UserUtil.java:383)
at cyder.ui.ConsoleFrame.setUUID(ConsoleFrame.java:1663)
at cyder.handlers.internal.LoginHandler.recognize(LoginHandler.java:500)
at cyder.handlers.internal.LoginHandler.autoCypher(LoginHandler.java:553)
at cyder.handlers.internal.LoginHandler.determineCyderEntry(LoginHandler.java:447)
at cyder.genesis.Cyder.main(Cyder.java:119)
[14-20-20] [SYSTEM_IO]: [JSON Saved] User was written to file: 7d7ebb00-81fe-329f-8a78-62f463d0b7da\Userdata.json
[14-20-20] [SYSTEM_IO]: File already existed in userspace: 14-20-20.stacktrace
[14-20-20] [EXCEPTION]:
Exception origin: com.google.common.base.Preconditions.checkArgument(Preconditions.java:131)
From line: 131
Trace: java.lang.IllegalArgumentException
at com.google.common.base.Preconditions.checkArgument(Preconditions.java:131)
at cyder.utilities.FileUtil.fileContentsEqual(FileUtil.java:141)
at cyder.utilities.UserUtil.userJsonBackupSubroutine(UserUtil.java:244)
at cyder.utilities.UserUtil.setUserData(UserUtil.java:176)
at cyder.utilities.UserUtil.logoutAllUsers(UserUtil.java:939)
at cyder.ui.ConsoleFrame.setUUID(ConsoleFrame.java:1667)
at cyder.handlers.internal.LoginHandler.recognize(LoginHandler.java:500)
at cyder.handlers.internal.LoginHandler.autoCypher(LoginHandler.java:553)
at cyder.handlers.internal.LoginHandler.determineCyderEntry(LoginHandler.java:447)
at cyder.genesis.Cyder.main(Cyder.java:119)
[14-20-20] [SYSTEM_IO]: [JSON Saved] User was written to file: 7d7ebb00-81fe-329f-8a78-62f463d0b7da\Userdata.json
[14-20-20] [SYSTEM_IO]: File already existed in userspace: 14-20-20.stacktrace
[14-20-20] [EXCEPTION]:
Exception origin: com.google.common.base.Preconditions.checkArgument(Preconditions.java:131)
From line: 131
Trace: java.lang.IllegalArgumentException
at com.google.common.base.Preconditions.checkArgument(Preconditions.java:131)
at cyder.utilities.FileUtil.fileContentsEqual(FileUtil.java:141)
at cyder.utilities.UserUtil.userJsonBackupSubroutine(UserUtil.java:244)
at cyder.utilities.UserUtil.setUserData(UserUtil.java:176)
at cyder.utilities.UserUtil.writeUser(UserUtil.java:101)
at cyder.utilities.UserUtil.setUserData(UserUtil.java:143)
at cyder.ui.ConsoleFrame.setUUID(ConsoleFrame.java:1670)
at cyder.handlers.internal.LoginHandler.recognize(LoginHandler.java:500)
at cyder.handlers.internal.LoginHandler.autoCypher(LoginHandler.java:553)
at cyder.handlers.internal.LoginHandler.determineCyderEntry(LoginHandler.java:447)
at cyder.genesis.Cyder.main(Cyder.java:119)
[14-20-20] [LOGIN]: [[AUTOCYPHER PASS]: 7d7ebb00-81fe-329f-8a78-62f463d0b7da]
[14-20-21] [SYSTEM_IO]: [JSON Saved] User was written to file: 7d7ebb00-81fe-329f-8a78-62f463d0b7da\Userdata.json
[14-20-21] [SYSTEM_IO]: Created temperatory file: 14-20-21.stacktrace
[14-20-21] [EXCEPTION]:
Exception origin: com.google.common.base.Preconditions.checkArgument(Preconditions.java:131)
From line: 131
Trace: java.lang.IllegalArgumentException
at com.google.common.base.Preconditions.checkArgument(Preconditions.java:131)
at cyder.utilities.FileUtil.fileContentsEqual(FileUtil.java:141)
at cyder.utilities.UserUtil.userJsonBackupSubroutine(UserUtil.java:244)
at cyder.utilities.UserUtil.setUserData(UserUtil.java:176)
at cyder.utilities.UserUtil.writeUser(UserUtil.java:101)
at cyder.utilities.UserUtil.setUserData(UserUtil.java:143)
at cyder.ui.ConsoleFrame.launch(ConsoleFrame.java:252)
at cyder.handlers.internal.LoginHandler.recognize(LoginHandler.java:513)
at cyder.handlers.internal.LoginHandler.autoCypher(LoginHandler.java:553)
at cyder.handlers.internal.LoginHandler.determineCyderEntry(LoginHandler.java:447)
at cyder.genesis.Cyder.main(Cyder.java:119)
[14-20-21] [SYSTEM_IO]: Userdata requested: RandomBackground
[14-20-21] [SYSTEM_IO]: Userdata requested: FullScreen
[14-20-21] [SYSTEM_IO]: Userdata requested: fullscreen
[14-20-21] [SYSTEM_IO]: Userdata requested: FullScreen [2x]
[14-20-21] [THREAD STARTED]: Console Printing Animation
[14-20-21] [SYSTEM_IO]: Userdata requested: OutputBorder
[14-20-21] [SYSTEM_IO]: Userdata requested: FullScreen
[14-20-21] [SYSTEM_IO]: Userdata requested: InputBorder
[14-20-21] [THREAD STARTED]: Console Input Caret Position Updater
[14-20-21] [SYSTEM_IO]: Userdata requested: OutputFill
[14-20-21] [SYSTEM_IO]: Userdata requested: InputFill
[14-21-33] [EOL]: Log completed, Cyder crashed unexpectedly: exit code: -9 External Stop, exceptions thrown: 4
